docker/nginx.conf

```diff
@@ -0,0 +1,36 @@
+user  nginx;
```

Comment:
why would we want a nginx config in our app repo, why can we not just publish the node module to npm and serve using unpkg as discussed.

Reply:
Because this is a pull request

Reply:
Would it not be preferable to have the Jenkins fetch the PR, build it and if build succeeds host the dist dir on some server ( as in; the Docker file, and nginx config you be part of the build repo and be reused across apps, instead of having to add this to every app repo and keep it al in sync and up to date, especially when there seem to be a lot of external dependencies)